fix metapost image inclusion (bug #457622)
upstream fix from Hans Hagen
---
 tex/context/base/supp-pdf.tex |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: context-2007.12.18/tex/context/base/supp-pdf.tex
===================================================================
--- context-2007.12.18.orig/tex/context/base/supp-pdf.tex	2007-12-24 09:36:00.000000000 +0100
+++ context-2007.12.18/tex/context/base/supp-pdf.tex	2007-12-24 09:36:15.000000000 +0100
@@ -912,7 +912,7 @@
 
 \def\doflushMPtext#1%
   {\edef\!!stringa{#1}%
-   \@EA\dodoflushMPtext\!!stringa\relax}
+   \expandafter\dodoflushMPtext\!!stringa\relax}
 
 \def\dodoflushMPtext
   {\afterassignment\dododoflushMPtext\let\nexttoken=}
@@ -921,10 +921,10 @@
   {\ifx\nexttoken\relax
      % done
    \else\ifx\nexttoken\char
-     \@EA\@EA\@EA\dodododoflushMPtext
+     \expandafter\expandafter\expandafter\dodododoflushMPtext
    \else
      {\nexttoken}%
-     \@EA\@EA\@EA\dodoflushMPtext
+     \expandafter\expandafter\expandafter\dodoflushMPtext
    \fi\fi}
 
 \def\dodododoflushMPtext
